### PR TITLE
mc: update to 4.8.27, split -skins

### PR DESCRIPTION
**Pull request by paper42**
Maintainer: N/A
Compile tested: aarch64, Turris MOX, OpenWrt 19.04
Run tested: aarch64, Turris MOX, OpenWrt 19.04

Description: 
* Fixes CVE-2021-36370 - server fingerprint isn't verified
* I also added a subpackage for skins - Options -> Appearance.

### CODE REVIEW

File: utils/mc/Makefile [hunk 1]
```diff
@@ -125,4 +125,19 @@ define Package/mc/conffiles
 /etc/mc/skins/default.ini
 endef
 
+define Package/mc-skins
+	TITLE:=Midnight Commander - a powerful visual file manager - skins
+	URL:=https://www.midnight-commander.org/
+	SECTION:=utils
+	CATEGORY:=Utilities
+	DEPENDS:=+mc
```
Comment (BKPepe) [inline utils/mc/Makefile:133]:
```suggestion
  TITLE:=Midnight Commander - a powerful visual file manager - skins
  URL:=https://www.midnight-commander.org/
  SECTION:=utils
  CATEGORY:=Utilities
  DEPENDS:=+mc
```

I think the **TITLE** might be so long that it does not display in `make menuconfig`, it needs to be verified.

Comment (paper42) [reply utils/mc/Makefile]: > I think the TITLE might be so long that it does not display in `make menuconfig`, it needs to be verified.

It fits for me.

Comment (BKPepe) [reply utils/mc/Makefile]: Ok, thanks for testing!

**Review by BKPepe — APPROVED** [review]

**Comment by paper42** [comment]
* dropped the 030-fix_cchar_t.patch, it is now fixed upstream: https://github.com/MidnightCommander/mc/commit/ba62dd32add2767d6d13f9abb82f43304418740f
* added /etc/mc/skins to conffiles, because default.ini from the base package is a conffile

**Comment by BKPepe** [comment]
cc: ex-maintainer @dibdot

**Comment by dibdot** [comment]
LGTM - successfully compiled & run tested with current master (x86-64, apu2) - thanks!

**Comment by dibdot** [comment]
@paper42 maybe you can fix a minor issue which comes up with the new release!? Whenever I start the internal editor (via F4) mc raises an error message that mc can't open a syntax file.

**Comment by paper42** [comment]
> maybe you can fix a minor issue which comes up with the new release!? Whenever I start the internal editor (via F4) mc raises an error message that mc can't open a syntax file.

sorry I missed that, I will submit a PR in a minute.

**Pull request merged**